README.md

```diff
@@ -1,231 +1,38 @@
-In such cases, users should define a custom `environment.yml` to manage their project's dependencies.
-We recommend starting with the [`environment.yml`](environment.yml) in this repository.
-The below modifications can help customize it for your needs:
-
-<details>
-
-1. Specifying versions for the listed packages, such as changing `python` to `python=3.6.5`.
-Versioned specification of your environment is critical to reproducibility and ensuring dependency updates do not break your pipeline.
-When first setting your package versions, you likely want to start with the latest versions available on the [conda-forge](https://anaconda.org/conda-forge/) channel, unless you have a reason to do otherwise.
-2. Adding other packages to your environment as required by your use case.
-Consider maintaining alphabetical sorting of packages in `environment.yml` to assist with maintainability.
-In addition, we recommend installing packages via pip, only if they are not available in the conda-forge channel.
-3. Add the `snorkel` package installation to your `environment.yml`, under the `- pip` section.
-Of course, we suggest versioning snorkel, which you can do via a release number or commit hash (to access more bleeding edge functionality)
-  ```yml
-    # Versioned via release tag
-    - git+https://github.com/HazyResearch/snorkel@v0.6.3
-    # Versioned via commit hash (commit hash below is fake to ensure you change it)
-    - git+https://github.com/HazyResearch/snorkel@7eb7076f70078c06bef9752f22acf92fd86e616a
-  ```
-Finally, consider versioning the `numbskull` and `treedlib` pip dependencies by changing `master` to their latest commit hash on GitHub.
-
-</details>
-
-### Development Environment
-
-This setup is intended for users who have cloned this repository and would like to access the environment for development.
-This approach installs the `snorkel` package in development mode, meaning that changes you make to the source code will automatically be applied to the `snorkel` package in the environment.
-
-```sh
-# From the root direcectory of this repo run the following command.
-conda env create --file=environment.yml
+To install Snorkel, run
 
-# Activate the conda environment (if using a version of conda below 4.4, use "source" instead of "conda")
-conda activate snorkel
-
-# Install snorkel in development mode
-pip install --editable .
+```bash
+pip3 install -U git+https://github.com/HazyResearch/snorkel#egg=snorkel
 ```
 
-### Additional installation notes
-
-<details>
+or 
 
-Snorkel can be installed directly from its GitHub repository via:
-
-```
-# WARNING: read installation section before running this command! This command
-# does not install any dependencies. It installs the latest master version but
-# you can change master to tag or commit
-pip install git+https://github.com/HazyResearch/snorkel@master
+```bash
+git clone https://github.com/HazyResearch/snorkel
+cd snorkel
+git checkout my-favorite-branch
```

Comment:
Just put `redux` here for now

Reply:
This generalizes to after redux is merged